var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -145,31 +146,31 @@ def setup_environment(self, spack_env, run_env):
         else:
             lib_dir = self.prefix.lib
 
-        run_env.set('ParaView_DIR', self.prefix)
-        run_env.set('PARAVIEW_VTK_DIR',
+        env.set('ParaView_DIR', self.prefix)
+        env.set('PARAVIEW_VTK_DIR',
                     join_path(lib_dir, 'cmake', self.paraview_subdir))
```

Comment:
@jrood-nrel Semantically the changes look good but flake is complaining about this line (need to adjust spacing). Other than that this looks good.